### ISSUE TITLE
Presets for colorblind users

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#57589]
### Is your feature request related to a problem? Please describe.

There are currently two presets of color themes: default and no bright background. Unfortunately there are no presets for colorblind users, giving them a considerable and unfair disadvantage.

### Solution you would like.

Two additional color themes since there is not just kind of being colorblind:
- Protanopia & Deuteranopia for red/green color blindness
- Tritanopia for blue/yellow color blindness
- And optionally something addressing the complete lack of color vision (grayscale)

That would help these users to play the game.

### Describe alternatives you have considered.

Using 24 bit color codes so you have a greater range of color. This would require a terminal which supports it but most already do. The greater range of color would make it considerably easier to find distinguishable colors.

**Comment by ZhilkinSerg** [comment]
Color templates only swap curses colors and do not change RGB values for any colors. Curses version users with vision differences probably already have color profiles in terminal software they use. Tiles version users can change RGB values using base_colors.json in their config folder. We do not provide any custom templates for base_colors.json.

**Comment by mryellow** [comment]
The default colours also feature #000 backgrounds with #fff fonts. This high level of contrast is very taxing on the eyes.
